core/services/ocrcommon/telemetry.go

```diff
@@ -431,3 +431,13 @@ func EnqueueEnhancedTelem[T EnhancedTelemetryData | EnhancedTelemetryMercuryData
 	default:
 	}
 }
+
+// getResultFloat64 will check the result type and force it to float64 or returns an error if the conversion cannot be made
+func getResultFloat64(task *pipeline.TaskRunResult) (float64, error) {
+	result, err := utils.ToDecimal(task.Result.Value)
+	if err != nil {
+		return 0, err
+	}
+	resultFloat64, _ := result.Float64()
```

Comment:
Whats the second return value here?

Reply:
It's a `bool` that indicates whether the float is exactly represented